starter-code/src/merge-sort.js

```diff
@@ -21,7 +21,9 @@ function sortHalves(left, right) {
   // array.slice() with no arguments is a trick to make a copy of the array
   // .concat is to smash all of the arrays together
   // ...maybe there's an ES6 way to do this?
-  return array.concat(left.slice()).concat(right.slice());
+  let merged = left.length ? [...array, ...left] : [...array, ...right];
```

Comment:
It's definitely not what was expected and I'm surprised it worked.
Merge sort should merge both halves, not single them out.

The equivalent ES6 code would have been `[...array, ...left, ...right]`.